api/v1alpha1/databaseclusterrestore_types.go

```diff
@@ -30,49 +30,32 @@ const (
 	BackupStorageAzure BackupStorageType = "azure"
 )
 
-type (
-	// RestoreState represents state of restoration.
-	RestoreState string
+// RestoreState represents state of restoration.
+type RestoreState string
 
-	// DatabaseClusterRestoreSpec defines the desired state of DatabaseClusterRestore.
-	DatabaseClusterRestoreSpec struct {
-		DatabaseCluster string        `json:"databaseCluster"`
-		DatabaseType    EngineType    `json:"databaseType"`
-		BackupName      string        `json:"backupName,omitempty"`
-		BackupSource    *BackupSource `json:"backupSource,omitempty"`
-	}
-	// BackupSource represents settings of a source where to get a backup to run restoration.
-	BackupSource struct {
-		Destination           string                     `json:"destination,omitempty"`
-		StorageName           string                     `json:"storageName,omitempty"`
-		S3                    *BackupStorageProviderSpec `json:"s3,omitempty"`
-		Azure                 *BackupStorageProviderSpec `json:"azure,omitempty"`
-		StorageType           BackupStorageType          `json:"storage_type"`
-		Image                 string                     `json:"image,omitempty"`
-		SSLSecretName         string                     `json:"sslSecretName,omitempty"`
-		SSLInternalSecretName string                     `json:"sslInternalSecretName,omitempty"`
-		VaultSecretName       string                     `json:"vaultSecretName,omitempty"`
-	}
-)
+// DatabaseClusterRestoreSpec defines the desired state of DatabaseClusterRestore.
+type DatabaseClusterRestoreSpec struct {
+	// DBClusterName defines the cluster name to restore.
+	DBClusterName string `json:"dbClusterName"`
```

Comment:
It's the same story here that clusterName sounds clear enough because restores can be applied against db clusters :). However, I'm okay with the proposed naming

Reply:
I only called it like this because it was the name we already used in the DatabaseClusterBackup and wanted to keep it consistent.